codeSnippets/snippets/autoreload/build.gradle

```diff
@@ -0,0 +1,36 @@
+buildscript {
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+
+apply plugin: 'kotlin'
+apply plugin: 'application'
+
+mainClassName = "io.ktor.server.netty.EngineMain"
+
+application {
+    // As an alternative way, uncomment this line to enable the development mode using the 'io.ktor.development' system property
+    // applicationDefaultJvmArgs = ["-Dio.ktor.development=true"]
+}
+
+sourceSets {
+    main.kotlin.srcDirs = [ 'src']
```

Comment:
I don't exactly know do we agree on the standard layout for new samples or not.

Reply:
do you mean directory structure or smth else?

Reply:
Directory structure

Reply:
Updated a structure 
![image](https://user-images.githubusercontent.com/38073144/109156179-96d59b80-7781-11eb-9c1b-7071434cbd5a.png)